src/styles/annotator/components/AdderToolbar.scss

```diff
@@ -1,129 +1,38 @@
-  // Adder, even when using Shadow DOM.
-  all: initial;
-
-  position: absolute;
-  box-sizing: border-box;
-  direction: ltr;
-  border-radius: var.$annotator-border-radius;
-
-  // Prevent the browser from selecting text in the adder itself during text
-  // selection.
-  //
-  // See https://github.com/hypothesis/product-backlog/issues/878
-  user-select: none;
-
-  // The adder toolbar has a distinctive, broad-spreading drop shadow
-  box-shadow: 0px 2px 10px 0px rgba(0, 0, 0, 0.25);
-
-  // Give the adder a very low opacity initially.  It will then fade-in when
-  // shown.
-  opacity: 0.05;
-
-  // Adder entry animation settings
-  animation-duration: $adder-transition-duration;
-  animation-timing-function: ease-in;
-  animation-fill-mode: forwards;
-
-  &--down.is-active {
-    animation-name: adder-fade-in, adder-pop-down;
-  }
-
-  &--up.is-active {
-    animation-name: adder-fade-in, adder-pop-up;
-  }
-}
-
-.AdderToolbar__arrow {
-  position: absolute;
-  z-index: 2;
-  color: var.$color-border;
-  fill: var.$color-background;
-
-  // Horizontal centering, part 1
-  left: 50%;
-
-  // Override `Icon` default 1em sizing
-  height: auto;
-  width: auto;
-
-  &--down {
-    // These transforms:
-    // 1) Translate -50% (left) on the X axis to complete horizontal centering
-    // 2) Translate up (Y) by 1px to eliminate a faint border line visible in Safari
-    transform: rotateX(180deg) translateX(-50%) translateY(1px);
-  }
-
-  &--up {
-    // Align at top edge of parent element...
-    top: 0;
-    // Translate on X axis -50% (left) to complete horizontal centering
-    // Translate -100% on Y axis (up) to position arrow above parent
-    transform: translate(-50%, -100%);
-  }
-}
-
-.AdderToolbar__badge {
-  // The badge should be vertically aligned with icons in other toolbar buttons
-  // and the label underneath should also align with labels in other buttons.
-  border-radius: var.$annotator-border-radius;
-  color: var.$color-text--inverted;
-  font-weight: bold;
-  // For positioning the number appropriately within the badge background
-  padding: 2px 4px;
-}
-
-.AdderToolbar__separator {
-  // Override border color to be a little darker
-  border-right: 1px solid var.$grey-4;
-}
-
-// The toolbar has full contrast when not hovered. When the toolbar is hovered,
-// the buttons are dimmed except for the one which is currently hovered.
-.AdderToolbar__actions:hover {
-  .AdderToolbar__button:not(:hover) {
-    color: var.$grey-semi;
-
-    .AdderToolbar__badge {
-      background-color: var.$grey-semi;
+@layer components {
+  // Main class for the root element of the "adder" toolbar that appears when the
+  // user makes a text selection.
+  .AdderToolbar {
+    // Reset all inherited properties to their initial values. This prevents CSS
+    // property values from the host page being inherited by elements of the
+    // Adder, even when using Shadow DOM.
+    all: initial;
+    direction: ltr;
```

Comment:
> Setting direction: ltr. I'm ignorant here, but believe direction is typically set as an HTML attribute, not a CSS rule?

See https://developer.mozilla.org/en-US/docs/Web/CSS/direction

Reply:
@esanzgar Sorry, yes, I know it _can_ be set as a CSS rule but I don't think it typically _is_. I'll look into this more.

Reply:
OK, meeting halfway here. Given that MDN's docs say:

> Note that text direction is usually defined within a document (e.g., with [HTML's dir attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir)) rather than through direct use of the direction property.

I'm going to move this from a CSS rule to an attribute on the relevant `div` element.